internal/pkg/cli/job_init.go

```diff
@@ -323,6 +336,21 @@ func jobTypePromptOpts() []prompt.Option {
 	return options
 }
 
+func (o *initJobOpts) dockerPlatform() (os, arch string, err error) {
+	os, arch = runtime.GOOS, runtime.GOARCH
+	if o.image == "" {
+		os, arch, err = o.dockerEngine.GetPlatform()
+		if err != nil {
+			return "", "", fmt.Errorf("get os/arch from docker: %w", err)
+		}
+	}
+	// Until we target X86_64 for ARM architectures, log a warning.
+	if arch == exec.ArmArch || arch == exec.Arm64Arch {
+		log.Warningf("Architecture type %s is currently unsupported.\nTo deploy, run %s\n", arch, "`DOCKER_DEFAULT_PLATFORM=linux/amd64 copilot svc deploy`")
```

Comment:
nit: so that the command also works for jobs.
```suggestion
		log.Warningf("Architecture type %s is currently unsupported.\nTo deploy, run %s\n", arch, "`DOCKER_DEFAULT_PLATFORM=linux/amd64 copilot deploy`")
```

Reply:
Removed "svc" from this warning in `svc_init` also.